docs/source/model_sharing.mdx

```diff
@@ -77,6 +77,8 @@ Converting a checkpoint for another framework is easy. Make sure you have PyTorc
 
 For example, suppose you trained DistilBert for sequence classification in PyTorch and want to convert it to it's TensorFlow equivalent. Load the TensorFlow equivalent of your model for your task, and specify `from_pt=True` so 🤗 Transformers will convert the PyTorch checkpoint to a TensorFlow checkpoint:
 
+<frameworkcontent>
+<tf>
```

Comment:
I fell a bit of the text above should be in the TF block.

Reply:
Good catch, I'll rearrange these! I thought I could set the `<tf>` block to come first, but it still displays the `<pt>` block first when I do that.